homework10/zanovo10/Colection/CollectionViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  CollectionViewController.swift
+//  zanovo10
+//
+//  Created by qeqwe on 20.11.2023.
+//
+
+import UIKit
+
+final class CollectionViewController: UIViewController {
+    
+    private lazy var collectionView: UICollectionView = {
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.minimumLineSpacing = 10
+        layout.minimumInteritemSpacing = 10
+        layout.sectionInset = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        layout.itemSize = CGSize(width: 150, height: 230)
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        return collectionView
+    }()
+    
+    private lazy var searchBar: UISearchBar = {
+        let bar = UISearchBar()
+        bar.frame = CGRect(x: 0, y: 80, width: 250, height: 30)
+        bar.showsCancelButton = true
+        bar.placeholder = "Poisk"
+        bar.sizeToFit()
+        return bar
+    }()
+    
+    private let sourse: [Photo] = Source.allPhotos()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupCollectionView()
+        view.addSubview(searchBar)
+        view.backgroundColor = .white
```

Comment:
```suggestion
        view.addSubview(searchBar)
        view.backgroundColor = .white
```
вынеси в сетап коллекции, или отдельный метол